scala-core/src/test/scala/com/baeldung/scala/durationsugar/DurationSugarTests.scala

```diff
@@ -0,0 +1,11 @@
+package com.baeldung.scala.durationsugar
+
+import org.scalatest._
+
+class DurationSugarTests extends FlatSpec with Matchers {
```

Comment:
Let's follow the naming convention for unit tests e.g) *UnitTest - see other packages.